README.md

```diff
@@ -71,38 +82,125 @@ Add the following content to the `.env` file:
 
-More details about the configuration with code examples can be found in the [wiki](https://github.com/boldare/openai-assistant/wiki/%F0%9F%A4%96-AI-Assistant#step-3-configuration).
+```typescript
+// chat.config.ts file
+import { AssistantConfigParams } from '@boldare/openai-assistant';
+import { AssistantCreateParams } from 'openai/resources/beta';
+
+// Default OpenAI configuration
+export const assistantParams: AssistantCreateParams = {
+  name: 'Your assistant name',
+  instructions: `You are a chatbot assistant. Speak briefly and clearly.`,
+  tools: [
+    { type: 'code_interpreter' }, 
+  ],
+  model: 'gpt-4-turbo',
+  temperature: 0.05,
+};
+
+// Additional configuration for our assistant
+export const assistantConfig: AssistantConfigParams = {
+  id: process.env['ASSISTANT_ID'],
+  params: assistantParams,
+  filesDir: './apps/api/src/app/knowledge',
+  toolResources: {
+    codeInterpreter: {
+      fileNames: [],
+    },
+  },
+};
+```
+
+
+More details about the configuration can be found in the [wiki](https://github.com/boldare/openai-assistant/wiki/%F0%9F%A4%96-AI-Assistant#step-3-configuration).
+
+#### What is this step for?
+From now you can run your application and call the assistant.
 
 
 ### Step 4: Function calling
 
+Function calling allows you to extend the assistant's capabilities with custom logic. **If you are not going to use function calling you can jump to: [Step 5: Testing](#step-5-testing).**
+
 Create a new service that extends the `AgentBase` class, fill the definition and implement the `output` method.
 
-- The `output` method is the main method that will be called when the function is invoked.
-- The `definition` property is an object that describes the function and its parameters.
+- The `output` method is the main method that will be called when the function is invoked by the assistant.
+- The `definition` property is an object that describes the function and its parameters so the assistant can understand how to call it.
 
 For more information about function calling, you can refer to the [OpenAI documentation](https://platform.openai.com/docs/assistants/tools/defining-functions).
 
 The instructions for creating a function can be found in the [wiki](https://github.com/boldare/openai-assistant/wiki/%F0%9F%A4%96-AI-Assistant#step-4-function-calling), while examples can be found in the [agents](apps/api/src/app/chat/agents) directory.
 
+#### What is this step for?
+
+If you've defined a function and the output method, you can now call it from the assistant just by asking him to do the action described in the function definition.
+
+
+### Step 5: Running the application and testing
+
+Run your application and this will allow you to test the assistant. 
+  
+  ```bash
+  # use this if you are using the repository:
+  npm run start:dev
+
+  # if you are using NestJS please check the npm scripts in the package.json file
```

Comment:
`if you are using your own NestJS application, please check the npm scripts in the package.json file`